addons/portal_rating/static/src/xml/portal_chatter.xml

```diff
@@ -5,14 +5,13 @@
     -->
     <t t-extend="portal.Composer">
         <t t-jquery="textarea" t-operation="inner"><t t-esc="widget.options['default_message'] ? _.str.trim(widget.options['default_message']) : ''"/></t><!-- need to be one line to avoid \t in textarea -->
-        <t t-jquery="form" t-operation="attributes">
-            <attribute name="t-attf-action">#{widget.options['force_submit_url'] ? widget.options['force_submit_url'] : '/mail/chatter_post'}</attribute>
+        <t t-jquery="button#chatter_composer_submit_btn" t-operation="attributes">
+            <attribute name="t-attf-data-action">#{widget.options['force_submit_url'] ? widget.options['force_submit_url'] : '/mail/chatter_post'}</attribute>
         </t>
-        <t t-jquery=".o_portal_chatter_composer_form input[name='csrf_token']" t-operation="after">
+        <t t-jquery=".o_portal_chatter_composer_input div.mb32" t-operation="before">
```

Comment:
please don't use utility classes like mb32 in t-jquery selectors, this will break very easily. If you need a more specific selector, add a new class on the div.